### ISSUE TITLE
The bible and commentary buttons should be removed from the reading plan screen

### ISSUE THREAD

**Issue by agrogers** [AndBible/and-bible#1548]
I dont really see the point of these buttons now. And i think it is a left over from an old version of AB. Can I remove them?

![image](https://user-images.githubusercontent.com/13920678/128656435-fe8c4cad-1f2b-4039-8c55-ec2ac0a66ac7.png)

**Comment by timbze** [comment]
@agrogers yes, those buttons need to be removed. I think the whole toolbar needs to change.
- I think it should have the back (up) button like other activities have
- But then we need a good way to switch reading plans, and go to another day in the reading plan. 
- I think a complete UI redesign would be good

**Comment by agrogers** [comment]
Yeah, I agree. I have been thinking about this too. Thought I would wait
until v4 is released before I add this one in 😁 I have seen some nice
reading plan layouts in other apps.
